dataflow_pipeline.py

```diff
@@ -0,0 +1,111 @@
+from __future__ import absolute_import
+
+import argparse
+import logging
+import re
+import timeit
+
+import apache_beam as beam
+from apache_beam.io import ReadFromText
+from apache_beam.io import WriteToText
+from apache_beam.metrics import Metrics
+from apache_beam.metrics.metric import MetricsFilter
+from apache_beam.options.pipeline_options import PipelineOptions
+from apache_beam.options.pipeline_options import SetupOptions
+import json
+
+from scripts.google_replicate import exec_google_copy
+
+try:
+    unicode  # pylint: disable=unicode-builtin
+except NameError:
+    unicode = str
+
+FILE_HEADERS = ["id", "filename", "md5", "size", "state", "acl", "project_id"]
+
+# global_config ={"token_path": "./gdc-token.txt",
```

Comment:
dead code, remove?

Reply:
fixed in d47a36c889171c98bbe27a87e89fb6d55d356f0f